libs/blog/articles/feature-article/src/lib/article-details-container/article-details-container.component.ts

```diff
@@ -40,5 +44,10 @@ export class ArticleDetailsContainerComponent {
 
   constructor() {
     this.articleDetailsStore.fetchArticleDetails(this.articleSlug);
+    effect(() => {
+      this.ariaLive = this.articleDetailsStore.isFetchArticleDetailsLoading()
+        ? 'polite'
+        : 'off';
+    });
```

Comment:
hmm first time seeing aria-live but shouldn't we show polite when loading has ended ? so it should be `!this.articleDetailsStore.isFetchArticleDetailsLoading() ? 'polite' : 'off'`

Reply:
i think @component attribute can access the component scope params (see button.component.ts) 
```
host: {
    '[class]': 'class()',
  },
```

Reply:
@jakNatonek that's cool, didn't know you could bind to @Component attributes like this :smile: I'll push a fix 

As for `aria-live'`, I've just tested it with a screen reader and it works exactly the same for the present solution, the change you suggested and keeping `aria-live="polite"`during the entire component lifecycle, making a "Loading" announcement. My intention was to prevent the screen reader from making unnecessary announcements after the content has been loaded from the API, but after reading your comment I dove into MDN and W3C documentation looking for guidelines to toggle the value `aria-live="polite"` on and off in cases like this and actually didn't find any.

With that being said, I'll remove the property binding and just keep it at `aria-live="polite"` for the entire component lifecycle.